ledger-core/virtual/execute/execute.go

```diff
@@ -892,8 +889,6 @@ func (s *SMExecute) stepAwaitSMCallSummary(ctx smachine.ExecutionContext) smachi
 		}).TryUse(ctx).GetDecision() {
 		case smachine.NotPassed:
 			return ctx.WaitShared(syncAccessor.SharedDataLink).ThenRepeat()
-		case smachine.Impossible:
```

Comment:
Why you've done that?

Reply:
Written here: https://insolario.slack.com/archives/CQQ0FSRNW/p1597317405311600